### PR TITLE
Added logging to detect hung async jobs in AdminClient.waitForCompletion

### PR DESCRIPTION
**Pull request by FelixGV**

### CODE REVIEW

File: src/java/voldemort/client/protocol/admin/AdminClient.java [hunk 1]
```diff
@@ -869,11 +869,25 @@ public String waitForCompletion(int nodeId,
 
             String description = null;
             String oldStatus = "";
+
+            // State to detect hung async jobs
+            long oldStatusTime = -1;
+            long timeSinceLastWarning = -1;
```
Comment (arunthirupathi) [inline src/java/voldemort/client/protocol/admin/AdminClient.java]: should this be called lastStatusReportTime

Comment (FelixGV) [inline src/java/voldemort/client/protocol/admin/AdminClient.java]: Hmm you're right. I chose the name hastily and it is misleading. How about ```lastHungStatusWarnTime``` ?

Comment (arunthirupathi) [inline src/java/voldemort/client/protocol/admin/AdminClient.java]: The variable is updated whenever status is reported, or warning is reported. So I am not sure the warn should be in the name. This is  a very minor detail, so I would leave it to you which way you want to go with.

Comment (FelixGV) [inline src/java/voldemort/client/protocol/admin/AdminClient.java]: I see what you mean. ```lastStatusReportTime``` makes sense, I'll go with that.

**Comment by FelixGV** [comment]
Superseded by #263. Closing.

**Pull request closed without merging**